### PR TITLE
Provision alerts for Loki Canary in Helm Chart.

### LINKED ISSUE #7284: [Helm] Provision Canary alerts.
```jsonnet
{
  name: 'canaries_alerts',
  rules: [
    {
      // End to end latency as seen by our canaries.
      alert: 'LokiCanaryLatency',
      expr: |||
        histogram_quantile(0.99, sum(rate(loki_canary_response_latency_seconds_bucket[5m])) by (le, namespace, cluster)) > 5
      |||,
      'for': '15m',
      labels: {
        severity: 'warning',
      },
      annotations: {
        message: |||
          {{ $labels.job }} is experiencing {{ printf "%.2f" $value }}s 99th percentile latency.
        |||,
      },
    },
  ],
},
```

### PR DESCRIPTION
**Pull request by jeschkies**
**What this PR does / why we need it**:
The Helm Charts install the Loki canary. It should also install the alerts for the canary so the users get the full value.

**Which issue(s) this PR fixes**:
Fixes #7284

**Special notes for your reviewer**:

**Checklist**
- [ ] Reviewed the `CONTRIBUTING.md` guide
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

File: production/helm/loki/src/alerts.yaml [hunk 1]
```diff
@@ -39,3 +39,14 @@ groups:
     for: 5m
     labels:
       severity: warning
+  name: 'loki_canaries_alerts',
+  rules:
+  - alert: 'LokiCanaryLatency',
```
Comment (jeschkies) [inline production/helm/loki/src/alerts.yaml:44]: @dannykopping do we also have alerts that verify read and write path of Loki are working? E.g. on `missing_entries`.

Comment (dannykopping) [reply production/helm/loki/src/alerts.yaml:44]: Not at present, but you can probably derive a few from Ed's _Loki Canary_ dashboard.
DM me if you don't have it

Do we have plans to bundle runbooks as part of Helm?

Comment (jeschkies) [reply production/helm/loki/src/alerts.yaml:44]: > Do we have plans to bundle runbooks as part of Helm?

Not for now.

> derive a few from Ed's Loki Canary dashboard

Ok, it feels this should just be part of the mixin.

Comment (dannykopping) [reply production/helm/loki/src/alerts.yaml:44]: Is there an issue I can upvote? I think runbooks would be easy to publish in the helmchart - we'd just need an nginx container and some markdown files really

Comment (jeschkies) [reply production/helm/loki/src/alerts.yaml:44]: I've just created https://github.com/grafana/loki/issues/7310

**Review by dannykopping — APPROVED** [review]
LGTM, nice

File: production/helm/loki/src/alerts.yaml (hunk 1 above)
Comment (trevorwhitney) [reply production/helm/loki/src/alerts.yaml:44]: what's the status of putting runbooks in grafana these days? the helm chart provides dashboards as a configmap you can mount to a grafana instance, is something similar possible or in the works for runbooks?

**Review by trevorwhitney — APPROVED** [review]

File: production/helm/loki/src/alerts.yaml (hunk 1 above)
Comment (towolf) [reply production/helm/loki/src/alerts.yaml:44]: > Not at present, but you can probably derive a few from Ed's _Loki Canary_ dashboard. DM me if you don't have it

@dannykopping is this dashboard posted anywhere? Is that the one mentioned in the "Loki Canary" blog post?

Comment (dannykopping) [reply production/helm/loki/src/alerts.yaml:44]: Which blogpost are you referring to @towolf?
We have [this](https://github.com/grafana/loki/blob/main/production/loki-mixin/dashboards/loki-canary-dashboard.libsonnet) as part of our jsonnet setup.

**Pull request merged**